app/src/main/java/com/nextcloud/talk/ui/bottom/sheet/ProfileBottomSheet.kt

```diff
@@ -0,0 +1,217 @@
+import com.nextcloud.talk.models.json.hovercard.HoverCardOverall
+import com.nextcloud.talk.ui.bottom.sheet.ProfileBottomSheet.AllowedAppIds.EMAIL
+import com.nextcloud.talk.ui.bottom.sheet.ProfileBottomSheet.AllowedAppIds.PROFILE
+import com.nextcloud.talk.ui.bottom.sheet.ProfileBottomSheet.AllowedAppIds.SPREED
+import com.nextcloud.talk.utils.ApiUtils
+import com.nextcloud.talk.utils.ConductorRemapping
+import com.nextcloud.talk.utils.bundle.BundleKeys
+import io.reactivex.Observer
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import org.parceler.Parcels
+
+private const val TAG = "ProfileBottomSheet"
+
+class ProfileBottomSheet(val ncApi: NcApi, val userEntity: UserEntity, val router: Router) {
+
+    private val allowedAppIds = listOf(SPREED.stringValue, PROFILE.stringValue, EMAIL.stringValue)
+
+    fun showFor(user: String, context: Context) {
+
+        ncApi.hoverCard(
+            ApiUtils.getCredentials(userEntity.username, userEntity.token),
+            ApiUtils.getUrlForHoverCard(userEntity.baseUrl, user)
+        ).subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
+            .subscribe(object : io.reactivex.Observer<HoverCardOverall> {
+                override fun onSubscribe(d: Disposable) {
+                }
+
+                override fun onNext(hoverCardOverall: HoverCardOverall) {
+                    bottomSheet(hoverCardOverall.ocs.data.actions, hoverCardOverall.ocs.data.displayName, user, context)
+                }
+
+                override fun onError(e: Throwable) {
+                    Log.e(TAG, "Failed to get hover card for user $user", e)
+                }
+
+                override fun onComplete() {
+                }
+            })
+    }
+
+    private fun bottomSheet(actions: List<HoverCardAction>, displayName: String, userId: String, context: Context) {
+
+        val filteredActions = actions.filter { allowedAppIds.contains(it.appId) }
+        val items = filteredActions.map { configureActionListItem(it) }
+
+        MaterialDialog(context, BottomSheet(LayoutMode.WRAP_CONTENT)).show {
+            cornerRadius(res = R.dimen.corner_radius)
+
+            title(text = displayName)
+            listItemsWithImage(items = items) { _, index, _ ->
+
+                val action = filteredActions[index]
+
+                when (AllowedAppIds.createFor(action)) {
+                    PROFILE -> openProfile(action.hyperlink, context)
+                    EMAIL -> composeEmail(action.title, context)
+                    SPREED -> talkTo(userId)
+                }
+            }
+        }
+    }
+
+    private fun configureActionListItem(action: HoverCardAction): BasicListItemWithImage {
+
+        val drawable = when (AllowedAppIds.createFor(action)) {
+            PROFILE -> R.drawable.ic_user
+            EMAIL -> R.drawable.ic_email
+            SPREED -> R.drawable.ic_phone
```

Comment:
This is a bit weird, can we not use the Talk icon aka `Q` similar to what is shown in the web?
The phone somehow makes me think it will do a call, although it just opens the chat.

Reply:
It's already like that in iOS, so we should align:

Web | Android | iOS
---|---|---
![Bildschirmfoto von 2021-11-05 11-11-04](https://user-images.githubusercontent.com/213943/140494848-d258daae-b389-4c0d-a563-5846957b5c79.png) | ![Screenshot_2021-11-05-11-11-14-387_com nextcloud talk2 qa](https://user-images.githubusercontent.com/213943/140494803-c651bb61-1f24-401a-b8e9-fa6e53f75c70.jpg) | ![2021-11-05 11-15-02-0](https://user-images.githubusercontent.com/213943/140495089-050703ce-9ead-4bf1-a13a-57eb9ba20719.png)

Reply:
![Screenshot_1636112853](https://user-images.githubusercontent.com/606665/140507145-8c291ef7-97c0-419e-8e4a-e387f55ad8ed.png)